### ISSUE TITLE
After global PC reboot, the container starts but the driver wont work anymore

### ISSUE THREAD

**Issue by MASSKAgithub** [yassine/soxy-driver#15]

**Comment by MASSKAgithub** [comment]
is not added to the store"
"bridge store not initialized.

and other things in logs

**Comment by yassine** [comment]
Did you start the driver container as well after the reboot? The driver has to be rebooted before the containers that uses the network.
I've tested this behavior (host reboot) many times before and can't reproduce it.

**Comment by MASSKAgithub** [comment]
I tried to restart, recreate networks even recreate the main knstance container and nothing works. Also when I rebooted I used the hard reboot from my host because the vps wasnt accessible anymore, when i reinstalled docker and the container all works back

**Comment by yassine** [comment]
are you saving/resolving some iptables rules between those reboots ?
(It shouldn't be the case)

**Comment by yassine** [comment]
could you please provide the outputs of following scenario after a reboot:
0. A dump of the 'docker network ls'
1. A full Dump of iptables rules for the nat & filter chains (iptables -t nat -v + iptables -t filter -v ) before starting any container
2. Start the driver container 
3. A full Dump of iptables rules for the nat & filter chains
4. Start the container being proxied
5. A full Dump of iptables the rules for the nat & filter chains
6. A full dump of the driver logs

**Comment by MASSKAgithub** [comment]
Im not saving anything, will do it when I come back to home.

**Comment by MASSKAgithub** [comment]
1.
root@bd-s1:~# docker network ls
NETWORK ID     NAME           DRIVER        SCOPE
9fbf30b3adda   R_US_1         soxy-driver   local
8193e278fa47   R_US_2         soxy-driver   local
7b71d0431f29   US_1           soxy-driver   local
de0888cc3e39   US_2           soxy-driver   local
c377200b4da6   US_3           soxy-driver   local
0edbad720b5a   US_4           soxy-driver   local
67600a5df58b   US_5           soxy-driver   local
ff9822a5bdc6   US_6           soxy-driver   local
4177eeb12b18   US_7           soxy-driver   local
cbc3c91e963f   US_8           soxy-driver   local
2ffc4465971c   US_9           soxy-driver   local
7ab4dcfe6f57   bridge         bridge        local
b79799fd138b   host           host          local
94b0e7f2cceb   none           null          local
20aba04afc10   soxy_network   soxy-driver   local

2. idk what command to run cuz each giveserror
3. already done
4. no idea how
5. not working the proxy not running cuz shit
when testing with your testing command:
* getaddrinfo(3) failed for check.torproject.org:443
* Couldn't resolve host 'check.torproject.org'
* Closing connection 0

6. seems like same as the 4 and 2 but same no idea
7. Is really too big
SO will give last ones

time="2022-10-06T18:19:13Z" level=debug msg="Received RevokeExternalConnectivity Request"
time="2022-10-06T18:19:13Z" level=debug msg="DeleteConntrackEntries purged ipv4:0, ipv6:0"
time="2022-10-06T18:19:13Z" level=warning msg="bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/c3de5fcaa6e05c0e0cebe8013438457b4bd049ad9e44ccdc7cc29bf8d889cfb6/ is not added to the store"
time="2022-10-06T18:19:13Z" level=debug msg="Received Leave Request %s @ %sc3de5fcaa6e05c0e0cebe8013438457b4bd049ad9e44ccdc7cc29bf8d889cfb6ff9822a5bdc67835791d111102843db41faf9fea300bbf9c0687fcd7ecdd4048"
time="2022-10-06T18:19:13Z" level=debug msg="Received DeleteEndpoint Request %s @ %sc3de5fcaa6e05c0e0cebe8013438457b4bd049ad9e44ccdc7cc29bf8d889cfb6ff9822a5bdc67835791d111102843db41faf9fea300bbf9c0687fcd7ecdd4048"
time="2022-10-06T18:19:13Z" level=debug msg="bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/c3de5fcaa6e05c0e0cebe8013438457b4bd049ad9e44ccdc7cc29bf8d889cfb6/ is not deleted from store"
time="2022-10-06T18:19:22Z" level=warning msg="bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/e508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963/ is not added to the store"
time="2022-10-06T18:19:22Z" level=debug msg="Received Join Request %s @ %se508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963ff9822a5bdc67835791d111102843db41faf9fea300bbf9c0687fcd7ecdd4048"
time="2022-10-06T18:19:22Z" level=debug msg="Received ProgramExternalConnectivity Request"
time="2022-10-06T18:19:22Z" level=warning msg="bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/e508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963/ is not added to the store"
time="2022-10-06T18:19:22Z" level=debug msg="Received EndpointInfo Request %s @ %se508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963ff9822a5bdc67835791d111102843db41faf9fea300bbf9c0687fcd7ecdd4048"
time="2022-10-06T18:19:22Z" level=debug msg="Received EndpointInfo Request %s @ %se508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963ff9822a5bdc67835791d111102843db41faf9fea300bbf9c0687fcd7ecdd4048"
time="2022-10-06T18:19:28Z" level=debug msg="Received RevokeExternalConnectivity Request"
time="2022-10-06T18:19:28Z" level=debug msg="DeleteConntrackEntries purged ipv4:0, ipv6:0"
time="2022-10-06T18:19:28Z" level=warning msg="bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/e508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963/ is not added to the store"
time="2022-10-06T18:19:28Z" level=debug msg="Received Leave Request %s @ %se508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963ff9822a5bdc67835791d111102843db41faf9fea300bbf9c0687fcd7ecdd4048"
time="2022-10-06T18:19:28Z" level=debug msg="Received DeleteEndpoint Request %s @ %se508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963ff9822a5bdc67835791d111102843db41faf9fea300bbf9c0687fcd7ecdd4048"
time="2022-10-06T18:19:28Z" level=debug msg="bridge store not initialized. kv object docker/network/v1.0/bridge-endpoint/e508c52665e301555f1bf9d49719513a319d16ae65b7ed0db0d2467f5d4b3963/ is not deleted from store"


yeah pls help verry annoying

**Comment by MASSKAgithub** [comment]
That's really borring, like need to recretated dozens of docker containers

**Comment by MASSKAgithub** [comment]
So after a rebooot iptables were getting ON with my hoster params

This fixed this alll big shit:

iptables -F
iptables-save


BRUH, so much time loss for a little shit
